rclcpp/include/rclcpp/node_interfaces/detail/node_interfaces_helpers.hpp

```diff
@@ -44,6 +44,10 @@ struct NodeInterfacesStorage
   : interfaces_(init_tuple<decltype(node), InterfaceTs ...>(node))
   {}
 
+  NodeInterfacesStorage()  // NOLINT(runtime/explicit)
```

Comment:
I think this NOLINT directive is not necesary. cpplint only enforces the usage of `explicit` for constructors with a single argument.

I tested it locally and the linters were happy with the NOLINT comment removed!

```suggestion
  NodeInterfacesStorage()
```

Reply:
Fixed in e65444629137633857995f86287118d213f0e7d1